articles/user-guide/language/expressions.md

```diff
@@ -0,0 +1,558 @@
+---------|----------|---------|---------
+ 5 | 2 | 2 | 1
+ 5 | -2 | -2 | 1
+ -5 | 2 | -2 | -1
+ -5 | -2 | 2 | -1
+
+Big integer division and modulus works the same way.
+
+Given any numeric expression, a new expression may be formed using the
+`-` unary operator.
+The new expression will be the same type as the constituent expression.
+
+Given any integer or big integer expression, a new expression of the same type
+may be formed using the `~~~` (bitwise complement) unary operator.
+
+## Boolean Expressions
+
+The two `Bool` literal values are `true` and `false`.
+
+Given any two expressions of the same primitive type, the `==`
+and `!=` binary operators may be used to construct a `Bool` expression.
+The expression will be true if the two expressions are equal, and false if not.
+
+Values of user-defined types may not be compared, only their unwrapped values can be compared. 
+For example, using the "unwrap" operator `!` (explained in detail at [Types in Q#](xref:microsoft.quantum.guide.types#access-anonymous-items-with-the-unwrap-operator)),
+
+```qsharp
+newtype WrappedInt = Int;     // Yes, this is a contrived example
+let x = WrappedInt(1);
+let y = WrappedInt(2);
+let z = x! == y!;             // This will compile and yield z = false.
+let t = x == y;               // This will cause a compiler error.
+```
+
+Equality comparison for `Qubit` values is identity equality;
+that is, whether the two expressions identify the same qubit.
+The state of the two qubits are not compared, accessed, measured, or modified
+by this comparison.
+
+Equality comparison for `Double` values may be misleading
+due to rounding effects.
+For instance, `49.0 * (1.0/49.0) != 1.0`.
+
+Given any two numeric expressions, the binary operators
+`>`, `<`, `>=`, and `<=` may be used to construct a new Boolean expression
+that is true if the first expression is respectively greater than, less than,
+greater than or equal to, or less than or equal to the second expression.
+
+Given any two Boolean expressions, the `and` and `or` binary operators
+may be used to construct a new Boolean expression that is true if both of
+(resp. either or both of) the two expressions are true.
+
+Given any Boolean expression, the `not` unary operator may be used to construct
+a new Boolean expression that is true if the constituent expression is false.
+
+## String Expressions
+
+Q# allows strings to be used in the `fail` statement and the `Log`
+standard function.
+
+Strings in Q# are either literals or interpolated strings.
+String literals are similar to simple string literals in most languages:
+a sequence of Unicode characters enclosed in double quotes, `"`.
+Inside of a string, the back-slash character `\` may be used to escape
+a double quote character, and to insert a new-line as `\n`, a carriage
+return as `\r`, and a tab as `\t`.
+For instance:
+
+```qsharp
+"\"Hello world!\", she said.\n"
+```
+
+The Q# syntax for string interpolations is a subset of the C# 7.0 syntax;
```

Comment:
This is another point where we could help out readers not already familiar with C#.